README.md

```diff
@@ -236,7 +233,7 @@ This example will showcase spooning state from a testnet contract into our local
 We will first start with the usual imports:
 
 ```rust
-use near_units::{parse_gas, parse_near};
+use near_units::parse_gas;
```

Comment:
I thought we refactored to use NearGas everywhere. Please, create a follow-up issue to finish the refactoring applying the same reasoning as we do now with near-token, i.e. breaking change to the API level in order to ensure strict typing.